packages/kbn-guided-onboarding/src/components/landing_page/guide_cards.constants.tsx

```diff
@@ -30,9 +32,15 @@ export interface GuideCardConstants {
 export const guideCards: GuideCardConstants[] = [
   {
     solution: 'search',
-    title: i18n.translate('guidedOnboardingPackage.gettingStarted.cards.appSearch.title', {
-      defaultMessage: 'Build an application on top of Elasticsearch',
-    }),
+    title: (
+      <FormattedMessage
+        id="guidedOnboardingPackage.gettingStarted.cards.appSearch.title"
+        defaultMessage="Build an application on {lineBreak} top of Elasticsearch"
+        values={{
+          lineBreak: <br />,
```

Comment:
I suggest using `<br/>` to break up the lines in a specific place because the cards are styled with the static width. So they wont grow or shrink in different screen sizes.

Reply:
That fixes the issue when only one word is on the second line 
> - Adds linebreaks to prevent one word dangling on the new line (as per @osmanis's https://github.com/elastic/kibana/pull/149528#issuecomment-1408780148)